Makefile Fixes / Updates

akutz opened the pull request:
This patch adds the variable `DRIVERS` to the build process where the command `$ DRIVERS=ebs make` would build libStorage with just the EBS storage driver and executor. Additionally, the `RBD_BUILD_TAGS` variable has been removed and the `test-rbd` target has been updated to use the new `DRIVERS` functionality.

codenrhoden reviewed (approved):
👍  👍  👍 

Yes, I'm that enthusiastic about this.

akutz committed:
Makefile Fixes / Updates

This patch adds the variable DRIVERS to the build process where the
command `$ DRIVERS=ebs make` would build libStorage with just the EBS
storage driver and executor. Additionally, the RBD_BUILD_TAGS variable
has been removed and the `test-rbd` target has been updated to use the
new DRIVERS functionality.
